hive/Dockerfile

```diff
@@ -63,7 +64,14 @@ RUN curl https://repo.stackable.tech/repository/packages/jmx-exporter/jmx_promet
     -o /stackable/jmx/jmx_prometheus_javaagent-0.16.1.jar && \
     chmod -x /stackable/jmx/jmx_prometheus_javaagent-0.16.1.jar
 
+# Logging
+RUN rm /stackable/hive/lib/log4j-slf4j-impl* && \
+    curl https://repo.stackable.tech/repository/packages/hive-jackson-dataformat-xml/${PRODUCT}/jackson-dataformat-xml-${JACKSON_DATAFORMAT_XML}.jar \
```

Comment:
It would be helpful to add a README.txt to https://repo.stackable.tech/repository/packages/hive-jackson-dataformat-xml which contains the URL where the jackson-dataformat-xml packages are mirrored from.

Many other packages also contain such a README file, e.g. https://repo.stackable.tech/repository/packages/hive/README.txt.

Reply:
done https://repo.stackable.tech/#browse/browse:packages:hive-jackson-dataformat-xml%2FREADME.txt